amgut/lib/data_access/ag_data_access.py

```diff
@@ -625,12 +628,18 @@ def AGGetBarcodeMetadataAnimal(self, barcode):
         return return_res
 
     def getAnimalParticipants(self, ag_login_id):
-        results = self._sql.execute_proc_return_cursor(
-            'ag_get_animal_participants', [ag_login_id])
-
-        return_res = [row[0] for row in results]
-        results.close()
-        return return_res
+        sql = """SELECT participant_name FROM ag_animal_survey
```

Comment:
The migration should be very simple, right? There are no new/different columns or anything so "old" surveys and "new" surveys have the same set of columns. So I don't think the migration should be a blocker for allowing editing. What else would be needed to permit pet survey editing? If there is little/nothing else, I think we should do it now.